docs/scripts.md

```diff
@@ -57,3 +57,26 @@ Entering shell ...
   "foo": "1"
 }
 ```
+
+Scripts can also have an optional description, which can be used in your `enterShell`.
+
+```nix title="devenv.nix"
+{ pkgs, ... }:
+
+{
+  scripts.silly-example.exec = ''
+    ${pkgs.curl}/bin/curl "https://httpbin.org/get?$1" | ${pkgs.jq}/bin/jq '.args'
+  '';
+  scripts.silly-example.description = ''simple example script'';
+  enterShell = ''
+      echo
+      echo 🦾 Helper scripts you can run to make your development richer:
+      echo 🦾
+      sed -e 's| |••|g' -e 's|=| |' <<EOF | column -t | sed -e 's|^|🦾 |' -e 's|••| |g'
+      ${lib.generators.toKeyValue {} (lib.mapAttrs (name: value: value.description) config.scripts)}
+      EOF
+      echo
+    '';
+}
+```
+
```

Comment:
Let's also show example output.

Reply:
Done. I've also added some more info in the examples section too.